searchform.php

```diff
@@ -6,7 +6,6 @@
 		<div class="SearchBar_Container">
 			<input class="SearchBar_Field" type="search" id="site-search" placeholder="<?php esc_html_e( 'Search the site &hellip;', 'hm-handbook' ); ?>" value="<?php the_search_query(); ?>" name="s"/>
 			<button class="SearchBar_Submit"><?php esc_html_e( 'Submit', 'hm-handbook' ); ?></button>
```

Comment:
not part of this PR but does the button need a `type="submit"`? or maybe just `submit_button('Submit', 'SearchBar_Submit')`